example.c

```diff
@@ -3,72 +3,139 @@
 #include <string.h>
 #include "linenoise.h"
 
+/* Functions your implementation needs to provide */
 
-void completion(const char *buf, linenoiseCompletions *lc) {
+/* linenoise_completion: Optional
+ * Provide completions to expand when Tab is pressed
+ */
+
+void linenoise_completion(const char *buf, linenoiseCompletions *lc) {
     if (buf[0] == 'h') {
         linenoiseAddCompletion(lc,"hello");
         linenoiseAddCompletion(lc,"hello there");
     }
+    if (!strcasecmp(buf, "/q")) {
+        linenoiseAddCompletion(lc,"/quit");
+    }
+    if (!strcasecmp(buf, "/c")) {
+        linenoiseAddCompletion(lc,"/count");
+    }
 }
 
-char *hints(const char *buf, int *color, int *bold) {
+/* linenoise_hints: Optional
+ * Provide hints to pop up as you type
+ */
+
+const char *world[] = { "World", "- Displays a traditional greeting"};
+const char *quit[] = { "/Quit", "- Exits this example"};
+const char *count[] = { "/Count", "- Prints the background counter"};
```

Comment:
The command completion right above uses lowercase for these …

Reply:
Done